Content.Server/DeviceNetwork/Systems/DeviceNetworkSystem.cs

```diff
@@ -219,70 +190,6 @@ private void SendToConnections(HashSet<DeviceNetworkComponent> connections, Devi
                     beforeEv.Uncancel();
             }
         }
-        #endregion
-
-        #region Component Setter Functions
-        public void SetReceiveFrequency(EntityUid uid, uint? frequency, DeviceNetworkComponent? device = null)
-        {
-            if (!Resolve(uid, ref device, false))
-                return;
-
-            if (device.ReceiveFrequency == frequency)
-                return;
-
-            if (!_networks.TryGetValue(device.DeviceNetId, out var deviceNet) || !deviceNet.UpdateReceiveFrequency(device.Address, frequency))
-                device.ReceiveFrequency = frequency;
-        }
-
-        public void SetTransmitFrequency(EntityUid uid, uint? frequency, DeviceNetworkComponent? device = null)
-        {
-            if (Resolve(uid, ref device, false))
-                device.TransmitFrequency = frequency;
-        }
-
-        public void SetReceiveAll(EntityUid uid, bool receiveAll, DeviceNetworkComponent? device = null)
-        {
-            if (!Resolve(uid, ref device, false))
-                return;
-
-            if (device.ReceiveAll == receiveAll)
-                return;
-
-            if (!_networks.TryGetValue(device.DeviceNetId, out var deviceNet) || !deviceNet.UpdateReceiveAll(device.Address, receiveAll))
-                device.ReceiveAll = receiveAll;
-        }
-
-        public void SetAddress(EntityUid uid, string address, DeviceNetworkComponent? device = null)
-        {
-            if (!Resolve(uid, ref device, false))
-                return;
-
-            if (device.Address == address)
-            {
-                device.CustomAddress = true;
-                return;
-            }
-
-            if (!_networks.TryGetValue(device.DeviceNetId, out var deviceNet) || !deviceNet.UpdateAddress(device.Address, address))
-            {
-                device.Address = address;
-                device.CustomAddress = true;
-            }
-        }
-
-        public void RandomizeAddress(EntityUid uid, string address, DeviceNetworkComponent? device = null)
-        {
-            if (!Resolve(uid, ref device, false))
-                return;
-
-            if (!_networks.TryGetValue(device.DeviceNetId, out var deviceNet) || !deviceNet.RandomizeAddress(device.Address, address))
-            {
-                var prefix = string.IsNullOrWhiteSpace(device.Prefix) ? null : Loc.GetString(device.Prefix);
-                device.Address = $"{prefix}{_random.Next():x}";
-                device.CustomAddress = false;
-            }
-        }
-        #endregion
```

Comment:
Why did you remove those methods?

Reply:
I'm guessing the reason was just because currently they're unused.

But they will be useful whenever devices are added that can actually have their address or frequencies set by users, as opposed to the current devices which just use random addresses on pre determined frequencies. So the functions should probably stay.

Reply:
I don't trust unused code. I added equivalent functions to address your concerns.